### PR TITLE
make the user field readonly and reduce the number of fields displayed on the user edit form

### PR DESCRIPTION
**Pull request by sewagodimo**

### CODE REVIEW

File: molo/commenting/models.py [hunk 1]
```diff
@@ -28,10 +35,6 @@ class MoloComment(MPTTModel, Comment):
 
     objects = MoloCommentManager()
 
-    class MPTTMeta:
```
Comment (codiebeulaine) [inline molo/commenting/models.py]: howcome this is removed?

File: molo/commenting/tests/test_admin.py [hunk 2]
```diff
@@ -383,3 +383,32 @@ def test_article_title_in_comment_view_can_contain_unicode(self):
...
+            content_object=article,
+            site=Site.objects.first(),
+            user=self.user,
+            comment='This is a comment',
+            parent=None,
+            submit_date=timezone.now())
+
+        response = self.client.get(
+            '/admin/commenting/molocomment/edit/' + str(comment.pk) + "/")
+        print(response)
```
Comment (codiebeulaine) [inline molo/commenting/tests/test_admin.py]: stray print

File: molo/commenting/tests/test_admin.py [hunk 3]
```diff
@@ -383,3 +383,32 @@ def test_article_title_in_comment_view_can_contain_unicode(self):
         response = self.client.get('/admin/commenting/molocomment/')
 
         self.assertContains(response, 'Test article 😴')
+
+    def test_article_comment_edit_fields(self):
+        '''
+        Test the articles can be edited on the admin interface
```
Comment (codiebeulaine) [inline molo/commenting/tests/test_admin.py]: incorrect doc string?

File: molo/commenting/models.py (hunk 1 above)
Comment (sewagodimo) [reply molo/commenting/models.py]: well that was an accident,

**Comment by codiebeulaine** [comment]
👍

**Pull request merged**